.travis.yml

```diff
@@ -45,7 +53,8 @@ jobs:
     - make vet
     - make lint
     - make fmt
-    - if [[ $(git diff --stat) != '' ]]; then echo 'Invalid formatting!' >&2; exit 1; fi
+    - git diff
```

Comment:
Why did you make the change?

Reply:
@ziflex to see `git diff` when pipelines fall and fix the problem.